general/scans/test/test_monoid.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+import numpy as np
+
+from mock import MagicMock
+from general.scans.monoid import Average, ListOfMonoids
```

Comment:
Unused import `ListOfMonoids`. Also, can we have some more test coverage for the monoid module?. It might be possible to add a unit test for `Sum` and `Polarisation` classes.

Reply:
I've added a unit test for the `Sum` class, but left `Polarisation` alone because it's not clear how I can independently verify what the expected behaviour is.